staging/src/k8s.io/mount-utils/mount_helper_common.go

```diff
@@ -53,7 +53,7 @@ func CleanupMountWithForce(mountPath string, mounter MounterForceUnmounter, exte
 	}
 	var notMnt bool
 	var err error
-	if !corruptedMnt {
+	if !mounter.canSafelySkipMountPointCheck() && !corruptedMnt {
```

Comment:
you only add the logic only to CleanupMountWithForce, which I only see nfs plugin uses it. So this skip logic only for nfs plugin?